README.md

```diff
@@ -1,2 +1,70 @@
-# wodin
-odin on the web
+# WODIN
+
+[odin](https://github.com/mrc-ide/odin) on the web
+
+## Introduction
+
+WODIN is a web application with a [Node.js](https://nodejs.org/en/) back end using [Express](http://expressjs.com/), 
+and a [Vue.js](https://vuejs.org/) front end. Both parts are written in [Typescript](https://www.typescriptlang.org/).  
+
+The WODIN application can be deployed with custom configuration to create teaching courses that allow students to author 
+and run model code using [odin](https://github.com/mrc-ide/odin).
+
+When the user runs a model in WODIN, odin will be invoked to provide executable JavaScript for odin model code. This 
+odin code may be written by the user or provided in config.
+
+WODIN configuration defines multiple 'apps', corresponding to teaching modules, which each provide functionality for 
+one of three types of model usage:
+- *basic* - to teach the basics of odin model definitions
+- *fit* - to fit a model to data
+- *stochastic* - to run a model stochastically
+
+Each app will have one of these three app types, as well as any further configuration required e.g. default odin code. 
+
+## Usage and configuration
+
+You can install dependencies, build and run the app locally in one step using `./scripts/build-and-run.sh`. The app will be available at http://localhost:3000 
+You may need to install TypeScript: `npm install -g typescript`
+
+WODIN will be deployable as a docker image which can be mounted with custom configuration as follows:
+
+### *.config.json
+
+Each config.json file defines config values for an app. The app will be available at `/apps/[appName]` for a config
+file named `[appName].config.json`
+
+### /files
+
+Sample data and other supporting files can be provided in any folder structure under `/files` and will be available at urls under `/files`
+
+### index.html 
+
+A root index page `index.html` should be provided which will be available at the root url `/` and should provide a front 
+page for all apps, including any description and explanation along with links to apps and data files.
+
+`index.html` may also use files included in the `/files` folder e.g. images or css.
+
+See the `/src/app/server/config` folder for example configuration, used in development. 
```

Comment:
I feel like might be nicer for the default config to live at the top level, like `scripts`, `config`, `app`? And I guess some conversation has happened about how to use this as an npm package, but I guess would just involve the config folder location being passed as a command line arg to the `npm build` task?